src/scheduler/misc.c

```diff
@@ -1720,17 +1720,14 @@ res_to_str_re(void *p, enum resource_fields fld, char **buf,
  *
  */
 int
-copy_attr_value(char **dest, char *src)
```

Comment:
This function isn't very different than string_dup().  It takes a string, allocates memory for it, and copies in the string.  This does pretty much the same thing, but frees any memory in the dest first.  That's kind of dangerous.  Just because the pointer isn't NULL doesn't mean it has any memory allocated to it.  If the pointer had been freed outside this function, you'll double free it in here.  I'd rather see you use string_dup().

Reply:
I have removed this function altogether.